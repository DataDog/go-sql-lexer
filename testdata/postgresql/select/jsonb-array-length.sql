SELECT jsonb_array_length(data->'tags') AS num_tags FROM products;
