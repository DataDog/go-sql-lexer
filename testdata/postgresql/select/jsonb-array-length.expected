[
    {
        "expected": "SELECT jsonb_array_length ( data -> ? ) FROM products",
        "statement_metadata": {
            "size": 14,
            "tables": [
                "products"
            ],
            "commands": [
                "SELECT"
            ],
            "comments": [],
            "procedures": []
        }
    },
    {
        "expected": "SELECT jsonb_array_length(data -> ?) FROM products",
        "normalizer_config": {
            "remove_space_between_parentheses": true
        }
    }
]